packages/account/src/Sections/Profile/LanguageSettings/language-settings.tsx

```diff
@@ -25,7 +25,14 @@ const LanguageSettings = observer(() => {
         switchLanguage(language_key);
     };
 
-    const allowed_languages: Record<string, string> = getAllowedLanguages(UNSUPPORTED_LANGUAGES);
+    let allowed_languages: Record<string, string> = getAllowedLanguages(UNSUPPORTED_LANGUAGES);
+
+    if (has_wallet) {
+        allowed_languages = Object.fromEntries(
+            Object.entries(allowed_languages).filter(([language_key]) => ['EN', 'AR'].includes(language_key))
```

Comment:
@wojciech-deriv  The plan is only to add AR on this initial integration as we've already released wallets to Morocco and it's a priority language. Next we'll add all the languages at once in [this card](https://app.clickup.com/t/20696747/WALL-4830) as they're all currently in translation progress, so we wouldn't need to gradually add on to the filtering logic and can just remove it altogether.

The reason we're doing AR separately is because it's the most arduous language which needs the most setup (support for RTL). Integrating the other languages wouldn't need extra effort besides removing these set filters